applications/leetcoin.md

```diff
@@ -107,81 +185,81 @@ Please also provide the GitHub accounts of all team members. If they contain no
-| 8. | Database               | Database to store all problem datasets (IPFS or MongoDB)                                                                                                                                              | 
-| 9. | API                    | Create an API with express, express router, mongoose and CORS                                                                                                                                         | 
-| 10. | Frontend user profile  | User profile page where they can edit their information and see their progress                                                                                                                        | 
-| 11. | Logging                | Server logging with morgan and winston                                                                                                                                                                | 
+|-------:|------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|    0a. | License                | Apache 2.0                                                                                                                                                                                            |
+|    0b. | Documentation          | We will provide the documentation for the repository code as well as a tutorial for publishing/designing a user-generated coding problem and test suites.                                             |
+|    0c. | Testing Guide          | Best practices for developing test suites to run code against and a FAQ.                                                                                                                              |
+|    0d. | Article                | Announcement blog post explaining how LeetCoin can be used for upgrading web3 developer skills, improve security and help with properly vetting dev core-team candidates to minimize technical risks. |
+|     1. | Design Document        | Description of architecture and interface                                                                                                                                                             |  
+|     2. | User Authentication    | Create a wallet based authentication system for problem builders as well as users with Web3 Onboard                                                                                                   |  
+|     3. | Online Judge           | Online judge that compiles Ink! contract code and runs it against test suites comparing the actual output with the expected ones based on the different test cases                                    |  
```

Comment:
`| 3. | Import/Export Data     | Create tools to import and export problem data using multer, csv-express, fast-csv                                                                                                                    | `

is it okay to exclude this one? We realized that since the first problem sets will be hand-crafted with other developers, import/export is not needed and we can populate the problems to our database using SQL. Also, since we use Postgres now as main storage it makes it easier to operate just with SQL.

The problems INSERT statements will be under the `database-seed.sql` or similar seed SQL file so that anyone can have access to them if they run locally.